package.json

```diff
@@ -6,7 +6,7 @@
     "video",
     "download"
   ],
-  "version": "0.7.22",
+  "version": "0.7.23",
```

Comment:
Don't include the new version number in the diff. This is usually done from the command `npm version`